scripts/start

```diff
@@ -38,7 +38,7 @@ fi
 
 # Configure Umbrel if it isn't already configured
 if [[ ! -f "${UMBREL_ROOT}/statuses/configured" ]]; then
-  NETWORK="${NETWORK:-mainnet}" "${UMBREL_ROOT}/scripts/configure"
+  NETWORK="${NETWORK:-mainnet}" BIND_ADDRESS="${BIND_ADDRESS:-0.0.0.0}" "${UMBREL_ROOT}/scripts/configure"
```

Comment:
You're using `BIND_ADDRESS` here but asking for `UMBREL_BIND_ADDRESS` in the docs:

```
sudo UMBREL_BIND_ADDRESS=127.0.0.1 ./scripts/start
```